### ISSUE TITLE
exercise 2 level 3 

### ISSUE THREAD

**Issue by yx420** [nus-cs2030/2223-s2#53]
hi may i know for the sentence "Let's now work on the loader service schedule. In the file level3.jsh, write a method", does it mean that we are provided with level3.jsh file? or we have to create it by ourself? if we have to create it by ourself then isit the same as creating .java file in the terminal?

**Comment by jorwong** [comment]
Hi, It means that you have to create one by yourself. You could create one using touch level3.jsh or vim level3.jsh and do a :wq! after editing.

**Comment by e0959170** [comment]
yup! for this lab we have to make everything ourselves. the .jsh file should js contain ur serveCruises methods and any other methods u wna use in it that are not in the classes
